web-wallet/src/style/main.css

```diff
@@ -126,9 +126,10 @@ svg {
     var(--secondary-color-variant-dark) 80%,
     transparent
   );
-  box-shadow: 0 0 0.5em
-    color-mix(in srgb, var(--secondary-color-variant-dark) 60%, transparent);
-  outline: 0 none;
+  border-width: var(--control-border-size);
+  outline: var(--control-border-size) solid
+    color-mix(in srgb, var(--secondary-color-variant-dark) 80%, transparent);
+  outline-offset: -0.125em;
```

Comment:
There is some weirdness with the `Switch` as well, as it doesn't seem to pick that style.

If you spot it at a glance fix the rule, otherwise open an issue for it as I think this is a regression we introduced in time.

Reply:
I created this for the switch issue as it needs some investigating https://github.com/dusk-network/rusk/issues/1537